hooks/command

```diff
@@ -188,14 +219,16 @@ EOF
   fi
 
   local COMMAND_BLOCKS=""
+  local ENTRYPOINT_BLOCK=""
   local COMMAND_BLOCK=""
   local ENVIRONMENT_BLOCK=""
   local VOLUMES_BLOCK=""
 
   for cmd in "$@"; do
+    ENTRYPOINT_BLOCK=$(get_docker_entrypoint)
     COMMAND_BLOCK=$(get_docker_command "$cmd")
     ENVIRONMENT_BLOCK=$(get_environment_block "$cmd")
-    VOLUMES_BLOCK=$(get_volumes_block "$cmd")
+    VOLUMES_BLOCK=$(get_volumes_block "$ENTRYPOINT_BLOCK")
```

Comment:
is this intended? i dont really understand whats going on here

Reply:
it actually looks like it never used the input. so i just replaced it with the input that i want

Reply:
doesn't that end up duplicating the block [here?](https://github.com/gantry-ml/simple-terraform-buildkite-plugin/pull/8/files#diff-94a101319341e0a2056fd83b6ec5656a8464b1af49f7192644132851e1256de6R251-R254)

Reply:
`get_volumes_block` doesn't actually spit `ENTRYPOINT_BLOCK` back out, so no. i can see how it's confusing tho